crawler/crawler.py

```diff
@@ -287,28 +288,50 @@ def crawler_worker(process_id, logfile, params):
 
     crawlutils.snapshot(**params)
 
+def docker_monitor(process_id, logfile, syncQ):
+    setup_logger('docker-monitor', logfile, process_id)
+    logger.info('*' * 50)
+    logger.info('Docker monitor #%d started.' % (process_id))
+    logger.info('*' * 50)
+
+    monitor = DockerMonitor(syncQ)
```

Comment:
This looks too docker specific to be in crawler.py but can't think of any other way. My issue is that there are other crawling modes besides OUTCONTAINER and we can crawl other container types. Just start a firefox browser, and you will see how the crawler catches the browser process as a container (they run it as actual linux containers).